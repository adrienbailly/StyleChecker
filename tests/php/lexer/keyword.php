abstract
and
as
break
case
catch
class
const
continue
default
die
de
doubleval
echo
else
elseif
endfor
endforeach
endif
endswitch
endwhile
exit
extends
false
final
floatval
for
foreach
function 
gettype
global
if
implements
import
include
include_once
instanceof
interface
intval
list
namespace
new
null 
or
print
print_r
private
protected
public
require 
require_once
return self
serialize
settype
static
strval
switch
throw
true
try
unserialize
unset
var
var_dump 
var_export
while
xor
