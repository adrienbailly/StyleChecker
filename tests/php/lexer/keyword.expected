(ABSTRACT,'abstract',1,0)
(NEWLINE,'\n',1,8)
(AND,'and',2,9)
(NEWLINE,'\n',2,12)
(AS,'as',3,13)
(NEWLINE,'\n',3,15)
(BREAK,'break',4,16)
(NEWLINE,'\n',4,21)
(CASE,'case',5,22)
(NEWLINE,'\n',5,26)
(CATCH,'catch',6,27)
(NEWLINE,'\n',6,32)
(CLASS,'class',7,33)
(NEWLINE,'\n',7,38)
(CONST,'const',8,39)
(NEWLINE,'\n',8,44)
(CONTINUE,'continue',9,45)
(NEWLINE,'\n',9,53)
(DEFAULT,'default',10,54)
(NEWLINE,'\n',10,61)
(DIE,'die',11,62)
(NEWLINE,'\n',11,65)
(DE,'de',12,66)
(NEWLINE,'\n',12,68)
(DOUBLEVAL,'doubleval',13,69)
(NEWLINE,'\n',13,78)
(ECHO,'echo',14,79)
(NEWLINE,'\n',14,83)
(ELSE,'else',15,84)
(NEWLINE,'\n',15,88)
(ELSEIF,'elseif',16,89)
(NEWLINE,'\n',16,95)
(ENDFOR,'endfor',17,96)
(NEWLINE,'\n',17,102)
(ENDFOREACH,'endforeach',18,103)
(NEWLINE,'\n',18,113)
(ENDIF,'endif',19,114)
(NEWLINE,'\n',19,119)
(ENDSWITCH,'endswitch',20,120)
(NEWLINE,'\n',20,129)
(ENDWHILE,'endwhile',21,130)
(NEWLINE,'\n',21,138)
(EXIT,'exit',22,139)
(NEWLINE,'\n',22,143)
(EXTENDS,'extends',23,144)
(NEWLINE,'\n',23,151)
(FALSE,'false',24,152)
(NEWLINE,'\n',24,157)
(FINAL,'final',25,158)
(NEWLINE,'\n',25,163)
(FLOATVAL,'floatval',26,164)
(NEWLINE,'\n',26,172)
(FOR,'for',27,173)
(NEWLINE,'\n',27,176)
(FOREACH,'foreach',28,177)
(NEWLINE,'\n',28,184)
(FUNCTION,'function',29,185)
(NEWLINE,'\n',29,193)
(GETTYPE,'gettype',30,194)
(NEWLINE,'\n',30,201)
(GLOBAL,'global',31,202)
(NEWLINE,'\n',31,208)
(IF,'if',32,209)
(NEWLINE,'\n',32,211)
(IMPLEMENTS,'implements',33,212)
(NEWLINE,'\n',33,222)
(IMPORT,'import',34,223)
(NEWLINE,'\n',34,229)
(INCLUDE,'include',35,230)
(NEWLINE,'\n',35,237)
(INCLUDE_ONCE,'include_once',36,238)
(NEWLINE,'\n',36,250)
(INSTANCEOF,'instanceof',37,251)
(NEWLINE,'\n',37,261)
(INTERFACE,'interface',38,262)
(NEWLINE,'\n',38,271)
(INTVAL,'intval',39,272)
(NEWLINE,'\n',39,278)
(LIST,'list',40,279)
(NEWLINE,'\n',40,283)
(NAMESPACE,'namespace',41,284)
(NEWLINE,'\n',41,293)
(NEW,'new',42,294)
(NEWLINE,'\n',42,297)
(NULL,'null',43,298)
(NEWLINE,'\n',43,302)
(OR,'or',44,303)
(NEWLINE,'\n',44,305)
(PRINT,'print',45,306)
(NEWLINE,'\n',45,311)
(PRINT_R,'print_r',46,312)
(NEWLINE,'\n',46,319)
(PRIVATE,'private',47,320)
(NEWLINE,'\n',47,327)
(PROTECTED,'protected',48,328)
(NEWLINE,'\n',48,337)
(PUBLIC,'public',49,338)
(NEWLINE,'\n',49,344)
(REQUIRE,'require',50,345)
(NEWLINE,'\n',50,352)
(REQUIRE_ONCE,'require_once',51,353)
(NEWLINE,'\n',51,365)
(RETURN,'return',52,366)
(SPACE,' ',52,372)
(SELF,'self',52,373)
(NEWLINE,'\n',52,377)
(SERIALIZE,'serialize',53,378)
(NEWLINE,'\n',53,387)
(SETTYPE,'settype',54,388)
(NEWLINE,'\n',54,395)
(STATIC,'static',55,396)
(NEWLINE,'\n',55,402)
(STRVAL,'strval',56,403)
(NEWLINE,'\n',56,409)
(SWITCH,'switch',57,410)
(NEWLINE,'\n',57,416)
(THROW,'throw',58,417)
(NEWLINE,'\n',58,422)
(TRUE,'true',59,423)
(NEWLINE,'\n',59,427)
(TRY,'try',60,428)
(NEWLINE,'\n',60,431)
(UNSERIALIZE,'unserialize',61,432)
(NEWLINE,'\n',61,443)
(UNSET,'unset',62,444)
(NEWLINE,'\n',62,449)
(VAR,'var',63,450)
(NEWLINE,'\n',63,453)
(VAR_DUMP,'var_dump',64,454)
(NEWLINE,'\n',64,462)
(VAR_EXPORT,'var_export',65,463)
(NEWLINE,'\n',65,473)
(WHILE,'while',66,474)
(NEWLINE,'\n',66,479)
(XOR,'xor',67,480)
(NEWLINE,'\n',67,483)
